standards/.pylintrc

```diff
@@ -88,7 +88,7 @@ function-rgx=([a-z][a-zA-Z0-9]*$)|([a-z][a-zA-Z0-9]*_$)
 method-rgx=([a-z][a-zA-Z0-9]*$)|([_]{2}[a-z]+[_]{2}$)|([a-z][a-zA-Z0-9]*_$)
 
 # Regular expression which should only match correct argument names
-argument-rgx=[a-z][a-zA-Z0-9]*$
+argument-rgx=[a-z][a-zA-Z0-9]*_?$
```

Comment:
I was testing it half day only with `dummy-variables-rgx=_$|dummy` and it was not working. So I guess pylint first makes `argument-rgx` check and adds if name is valid or not, if variable is not used it checks `dummy-variables-rgx`. Also this helped: http://stackoverflow.com/questions/19395165/pylint-isnt-ignoring-dummy-variables